server/email-notification.js

```diff
@@ -0,0 +1,70 @@
+require('dotenv/config');
+const format = require('date-fns/format');
+const { zonedTimeToUtc } = require('date-fns-tz');
+const db = require('./db.js');
+const sgMail = require('@sendgrid/mail');
+const formatDistanceStrict = require('date-fns/formatDistanceStrict');
+const { utcToZonedTime } = require('date-fns-tz');
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const checkTime = time => {
+  if (time === '6 hours' || time === '5 hours' || time === '4 hours' || time === '3 hours' || time === '2 hours' || time === '1 hours' || time === '0 hours') return true;
+  return false;
+};
+
+const todayUTC = zonedTimeToUtc(format(new Date(), 'yyyy-MM-dd HH'), Intl.DateTimeFormat().resolvedOptions().timeZone);
+const sql = `
+    select "title",
+           "description",
+           "timestamp",
+           "origin",
+           "destination",
+           "notification",
+           "sent",
+           "email",
+           "eventId"
+      from "events"
+     where "userId" = $1
```

Comment:
You should be able to select the correct events from the database with a `where` clause so you don't have to filter them in your JavaScript.

https://www.postgresql.org/docs/12/functions-datetime.html